docs/source/tutorial_quick.rst

```diff
@@ -79,7 +79,14 @@ including a full list of `tasks <http://parl.ai/docs/tasks.html>`_ and
 `instructions <http://parl.ai/docs/tutorial_basic.html#training-and-evaluating-existing-agents>`_
 on specifying arguments for training and evaluation (like the ``-t <task>`` argument used here).
 
-Let's begin again by printing the first few examples.
+First, we need to install some dependencies to build the twitter dataset.
+
+.. code-block:: bash
+
+  # install necessary dependencies
+  pip3 install emoji unidecode
```

Comment:
I'm still confused. If they need to be in requirements.txt, let's put them there. I'm just surprised. This tutorial shouldn't have any extra installation steps though.

Reply:
Let's just add it to requirements.txt and move on.